packages/webapp/src/pages/Environment/Settings.tsx

```diff
@@ -988,6 +1058,111 @@ export const EnvironmentSettings: React.FC = () => {
                                 </form>
                             </div>
                         </div>
+                        <div>
```

Comment:
To see with @bastienbeurier but I feel like it would be better for the UX to clearly state that the account has no access to this feature instead of clicking edit, pasting something, and then getting an error.
You can return the flag in the `/meta`.

Reply:
will check with @bastienbeurier

Reply:
as per our discussion with @bastienbeurier: yes this would be a better UX but let's ship it like this for now and we will work on redesigning the env settings page including clearly stating that this option is paid only